app/controllers/customer_items_controller.rb

```diff
@@ -0,0 +1,8 @@
+class CustomerItemsController < ApplicationController
+  def create
+    customer_item = CustomerItem.new(customer_id: params[:id], item_id: params[:item_id])
+    customer_item.save
+
```

Comment:
on ICs we don't expect you to do any error handling, so you can use `.create` instead if you'd like